### PR TITLE
Add viv to showcase

### LINKED ISSUE #4941: Add viv/avivator to deck.gl showcase page
I was talking to @ajduberstein and we agreed that the [Viv project](https://github.com/hms-dbmi/viv) and the corresponding [avivator viewer](http://avivator.gehlenborglab.org/?image_url=https://viv-demo.storage.googleapis.com/Vanderbilt-Spraggins-Kidney-MxIF.ome.tif) show a really great non-geospatial use case of deck.gl in the biomedical realm. I think it would be a great project to add to the showcases page in the documentation.

cc @ilan-gold. If you're interested, do you have a thumbnail, couple sentences of descriptive text, and a preferred link?

### PR DESCRIPTION
**Pull request by kylebarron**
### Change List

- Add viv to showcase. Closes #4941 

Questions:

cc @ilan-gold @ngehlenborg

- Since it's used as a thumbnail, I took the second image from #4941 and resized it to about 820x820px. It's still 470KB, which is 3x larger than the other thumbnails, so maybe we'll want to compress it further.
- Do you want an application link plus a source code link? I added an application link below but feel free to change it. Do you want a "By HMS/gehlenborg lab" or blog post link? Check out the existing entries for reference: https://deck.gl/showcase

### CODE REVIEW

**Comment by Pessimistress** [comment]
Have you tested the website? The text blob seems a little long.

**Comment by kylebarron** [comment]
I tried but got some missing module errors (after `yarn bootstrap; cd website; yarn; yarn start;`) and figured I'd take time later to try building the website

File: website/showcase.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "Viv",
+    "url": "http://avivator.gehlenborglab.org/?image_url=https://viv-demo.storage.googleapis.com/Vanderbilt-Spraggins-Kidney-MxIF.ome.tif",
+    "image": "/images/showcase/viv.jpg",
+    "links": {
+      "Source code": "https://github.com/hms-dbmi/viv"
```
Comment (ngehlenborg) [inline website/showcase.json:7]:
```suggestion
      "Source code": "https://github.com/hms-dbmi/viv",
      "By Gehlenborg Lab/Harvard Medical School": "http://gehlenborglab.org/"
```

**Review by ngehlenborg — APPROVED** [review]
If the text blob is too long, you can cut the "The development of ... Harvard University." sentence.

**Comment by kylebarron** [comment]
I'll try testing the website locally again this evening

**Review by Pessimistress — APPROVED** [review]

**Comment by kylebarron** [comment]
Was able to test it locally:

image

**Pull request merged**